pan3d/dataset_builder.py

```diff
@@ -319,6 +319,41 @@ def t_index(self, t_index: int) -> None:
                 self._viewer._time_index_changed()
                 self._viewer._mesh_changed()
 
+    @property
+    def t_num(self) -> int:
```

Comment:
t_size, t_len ?

Reply:
I settled for `t_size`, seems easier to identify